### DISCUSSION TITLE
Solution for putting the scenery files on another drive and accessing them with XP11 or XP12

### DISCUSSION THREAD

**Discussion by dpdurst** [kubilus1/autoortho#238] [Show and tell]
I've spent a few hours working through this on my Win 11 machine and XP12, I believe since it works here it will work in Win 10 and XP11 or XP12 as well. This worked for me,  I've only tested this with 0.5.0.x software

What this does is allow you to put the XP scenery files you download with AO on another drive and access them so that you do not run out of space on the drive that XP is installed in.

So here we go. I'll use E: drive as my XP install (and my AutoOrtho install location) and D: drive as the scenery file location

So your XP is installed in E: drive yet you have only 5 GB of space left and you want to run Auto Ortho, yet you have a D: drive with 100GB of space and want to use that to store the scenery files for AutoOrtho

You have 2 options here, you can install AO onto the D: drive and create a folder called Custom Scenery or you can install AO on any other drive and create on D: drive the folder called Custom Scenery (Its case sensitive and must be just like I have shown)

Once you have done that you can now start up AO and configure how you want, but point the X-Plane scenery dir to the Custom Scenery folder at the new location as shown below, once configured press the Save button and then select the scenery you want to download and let it all install

![ao1](https://github.com/kubilus1/autoortho/assets/8261471/0d7db6d7-108b-4090-9b09-1437ee3027fa)

Once the scenery has been installed you can press the Run button and make sure the folder links are created in the Custom Scenery folder you defined in the setup above and then shut down AO

Now you are going to take the folders AO created under D:\Custom Scenery -  yAutoOrtho_Overlays, z_ao_sa and z_autoortho and create shortcuts of them in your XP11 or XP12 Custom Scenery folder

![ao2](https://github.com/kubilus1/autoortho/assets/8261471/5bd0c9ca-b6a9-42bb-8b4c-a7c031da1027)

![shortcuts](https://github.com/kubilus1/autoortho/assets/8261471/95ef0cb2-3932-449d-9945-d44c3e5ee159)

Next Startup AO and select Run and then startup XP and once you get to the main menu shut down XP so that it then creates the entries in your scenery.ini file as follows

SCENERY_PACK D:\Custom Scenery\yAutoOrtho_Overlays/
SCENERY_PACK D:\Custom Scenery\z_ao_sa/
SCENERY_PACK D:\Custom Scenery\z_autoortho/

Next move these to the bottom under the Global Airports entry and save

Startup AO (if you shut it down) and/or Startup XP and AutoOrtho should start streaming the files to Xplane

![ao and xp](https://github.com/kubilus1/autoortho/assets/8261471/cfda97b0-2e3c-4abc-a5d7-6ef0143b2ed6)

**Comment by rtc79clm** [comment]
Thank you DP, this installation process allowed me to update the scenery_pack file. I discovered I have missing tiles and need to delete the installed files and download again.
